Documentation/configuring-hive-metastore.md

```diff
@@ -35,20 +35,20 @@ Use [metastore-storage.yaml][metastore-storage-config] as a template and adjust
 By default to make installation easier Metering configures Hive to use an embedded Java database called [Derby](https://db.apache.org/derby/#What+is+Apache+Derby%3F), however this is unsuited for larger environments or metering installations with a lot of reports and metrics being collected.
 Currently two alternative options are available, MySQL and PostgreSQL, both of which have been tested with the metering-operator.
 
-There are 4 configuration options you can use to control the database used by Hive metastore: `url` , `driver` , `username` , and `password`.
+There are 4 configuration options you can use to control the database used by Hive metastore: `url` , `driver` , `username` , `secretName` and `password`.
 
-Using MySQL:
+To use MySQL create a secret in your namespace and then add the following to your Metering CR:
 
 ```
-spec:
-  hive:
-    spec:
-      config:
-        db:
-          url: "jdbc:mysql://mysql.example.com:3306/hive_metastore"
-          driver: "com.mysql.jdbc.Driver"
-          username: "REPLACEME"
-          password: "REPLACEME"
+
```

Comment:
why take out the `spec:`

Reply:
Yeah, revert this - let's continue using the top-level spec in our YAML outputs (also applies to the following Posgresql documentation.)